.eslintcache

```diff
@@ -0,0 +1 @@
fConfig":"17cvli7","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/test/utils.spec.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/main.development.js":{"size":7894,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/main.development.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/postinstall.js":{"size":2053,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/postinstall.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/server.js":{"size":1352,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/server.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.base.js":{"size":1124,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.base.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.build.js":{"size":198,"mtime":1474230092000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.build.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.development.js":{"size":1643,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.development.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.eslint.js":{"size":86,"mtime":1474230092000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.eslint.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.main.js":{"size":707,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.main.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.mocha.js":{"size":355,"mtime":1476556641000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.mocha.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.renderer.js":{"size":1852,"mtime":1474230092000,"hashOfConfig":"1o8rflt","results":{"filePath":"/Users/amila/Documents/Projects/popcorn-time-desktop/webpack.config.renderer.js","messages":[],"errorCount":0,"warningCount":0}}}
```

Comment:
Whoops 😂 I gitignored the cache in one branch, switched to this branch and committed. Forgot that it wasnt ignored in this branch. Don't think it will work.